src/Particular.CodeRules/ConfigureAwaitAnalyzer/ConfigureAwaitAnalyzer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections.Immutable;
+
+namespace Particular.CodeRules.ConfigureAwaitAnalyzer
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class ConfigureAwaitAnalyzer : DiagnosticAnalyzer
```

Comment:
ConfigureAwaitAnalyzer.ConfigureAwaitAnalyzer? i think the namespace is redundant?